meson.build

```diff
@@ -111,6 +111,7 @@ endif
 prefixdir_noslash = '/' + prefixdir.strip('/')
 bindir = prefixdir / get_option('bindir')
 sbindir = prefixdir / (split_bin ? 'sbin' : 'bin')
+sbin_to_bin = split_bin ? '../bin/' : ''
```

Comment:
Why is this needed? Wouldn't bindir as before suffice?

Reply:
`install_symlink()` does not make the path relative.
Previously, we used `ln -sfr`, then the path was handled automatically and independently of `split_bin`.
But with this change, we need to do that manually.

Reply:
Honestly, I think we should file a meson issue for relative symlink support and wait to switch over until it's available. Relying on manually created relative links sounds like an accident to happen…